foundations/foundation-config/src/main/java/org/apache/servicecomb/config/spi/PlatformAdaptor.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.servicecomb.config.spi;
+
+/**
+ * 
+ * To adapte different Platforms 
+ */
+public interface PlatformAdaptor {
```

Comment:
1. Maybe we can have a more specific name, e.g. ConfigCenterAddressFinder
2. This implementation has coupled config module with config center module & service center module. It's not very good. I think this is config center, service center specific features. Or you need also to implement config-apollo and other modules
3. If you are using SPI, it's better to add an getOrder method and can resolve conflicts easier.